packages/@guardian/source-react-components-development-kitchen/src/toggle-switch/ToggleSwitch.tsx

```diff
@@ -82,12 +87,15 @@ export const ToggleSwitch = ({
 	defaultChecked,
 	cssOverrides,
 	platform = 'web',
+	disabled = false,
 	onClick = () => undefined,
 	...props
 }: ToggleSwitchProps): EmotionJSX.Element => {
 	const buttonId = id ?? generateSourceId();
 	const labelId = descriptionId(buttonId);
-	const isChecked = (): boolean => {
+	const isChecked = (disabled: boolean): boolean => {
```

Comment:
Can we get this directly from the prop rather than passing it as a param?

```suggestion
	const isChecked = (): boolean => {
```